src/esp/assets/Asset.cpp

```diff
@@ -14,12 +14,11 @@ using namespace Corrade::Utility::String;
 AssetInfo AssetInfo::fromPath(const std::string& path) {
   AssetInfo info{AssetType::UNKNOWN, path};
 
-  if (endsWith(path, "semantic_quad_mesh.ply")) {
-    info.type = AssetType::FRL_INSTANCE_MESH;
```

Comment:
Should we also remove `FRLInstanceMeshData.*`?  This was the only place that code could have possibly be run IIRC.

Reply:
Good point, I think so too.

Reply:
![image](https://user-images.githubusercontent.com/931093/64358338-251cf700-cfbb-11e9-8ee0-4e3e374ce712.png)

The latest "FRL_INSTANCE_MESH", which has the name "mesh_semantic.ply" and can be actually handled (and it is handled) by the AssetType::INSTANCE_MESH (see the picture above).

Is there any different (in terms of rendering, model training etc.) when we use FRL_INSTANCE_MESH and INSTANCE_MESH? Any exclusive functionalities designed for each type? If there is no difference, I think we can deprecate one (e.g., FRL_INSTANCE_MESH).

Thanks for rising up this issue. Yes, even if we decide to deprecate one, we can ship it in some other PR.

Reply:
The `FRL_INSTANCE_MESH` was for that initial scan of the apartment.  For replica, I rewrote the instance mesh exporter to be similar to the MP3D format so that we could load it via generic instance mesh.  The replica meshes do not get loaded by `FRL_INSTANCE_MESH`/ can't get loaded as that type.

Reply:
@erikwijmans : Good to know. Then we never ever use the "initial scan" in the released replica dataset. Correct?
Shall we just retire it? (I should revert my changes again.) 
I remembered you suggested we did not have to support the old replica format any more, and we agreed in the group conversation a couple of weeks ago.

Reply:
Yeah, that scan was never publicly release and I don't think anyone needs to load it for any reason.

Reply:
Somehow, I was under the impression we had already deprecated it a while back... In any case, agree, there should be a separate PR to deprecate it fully and avoid any future confusion like the above.